Modules/_threadmodule.c

```diff
@@ -178,7 +245,6 @@ static PyGetSetDef ThreadHandle_getsetlist[] = {
 
 static PyMethodDef ThreadHandle_methods[] =
 {
-    {"detach", (PyCFunction)ThreadHandle_detach, METH_NOARGS},
```

Comment:
I do hope nothing is depending on this, it's _supposed_ to be a very internal implementation detail.  we'll find out.  :)